packages/ckb-sdk-core/src/generateRawTransaction.ts

```diff
@@ -18,7 +20,9 @@ const generateRawTransaction = async ({
   safeMode: boolean
   cells?: CachedCell[]
   deps: DepCellInfo
-}) => {
+  capacityThreshold?: bigint | string | number
```

Comment:
So we still accept `number` as capacity?

Reply:
`number` is removed in https://github.com/nervosnetwork/ckb-sdk-js/pull/382/commits/3af1bfc9d0f2fdde4d2ff8a0a53bdf2c3d885a34